.github/workflows/ci.yml

```
@@ -15,7 +15,6 @@ jobs:
       fail-fast: false
       matrix:
         julia-version:
-          - "1.3"
           - "1"
```

I'd still run CI on the oldest supported version:
```suggestion
          - "1.6"
          - "1"
```